### ISSUE TITLE
BrAPI / Germinate update 

### ISSUE THREAD

**Issue by Don-Isdale** [plantinformatics/pretzel#423]
Background :  #372

---

#### Update the BrAPI / Germinate interface to match with server updates

+ [x] re-test BrAPI connectivity
Testing shows connectivity issue; addressed by the following sub-issue 425. 

---

Sub-issues :

- #425
- #426
- #427
- #429
- #430
---

**Comment by Don-Isdale** [comment]
#### Known issues completed

The 3 sub-issues which were reported and discovered in testing are completed.

This screen shot shows the successful result of testing, on local.

![Screenshot from 2024-11-22 11-58-11](https://github.com/user-attachments/assets/6e010016-939a-430a-ad6b-73c4022496e4)

**Comment by Don-Isdale** [comment]
#### Further issues completed

Sub-issues #429 and #430 were added, and are now complete and tested, so this issue can be re-closed.

**Comment by Don-Isdale** [comment]
#### Further issues completed

Sub-issues #429 and #430 were added, and are now complete and tested.
Screen-shots of the test results are shown in #430, showing successful access to the largest of the 3 datasets on the Germinate test server;  the other 2 datasets were successfully tested earlier as shown in the screenshots in the preceding [comment](https://github.com/plantinformatics/pretzel/issues/423#issuecomment-2492693319),
and also in screenshots posted in Teams channel :
```
Screenshot from 2024-11-26 11-06-50.png

Screenshot from 2024-11-27 10-45-31.png

Screenshot from 2024-11-27 10-52-19.png
Screenshot from 2024-11-27 10-51-47.png
```

This issue will now be re-closed.

**Comment by Don-Isdale** [comment]
#### Released as v3.2.0

The work described by this issue is merged to develop and master,  (PR #432, #435),  included in release v3.2.0, and deployed to agg.

All sub-issues and this head issue are completed and closed. Released as v3.2.0.

---
![Screenshot from 2024-11-29 17-50-55](https://github.com/user-attachments/assets/d921b7e8-e902-4fba-96c0-310d1d5894a5)
